WordPress/Classes/Extensions/NSFetchedResultsController+Helpers.swift

```diff
@@ -17,10 +17,14 @@ extension NSFetchedResultsController
         return indexPath.row == sections[indexPath.section].numberOfObjects - 1
     }
 
+/*
```

Comment:
Can we 💥 the nuked code?

I think it'll make the changes in the PR more evident for reviewers as well.